cluster/lib/logging.sh

```diff
@@ -114,6 +120,11 @@ kube::log::usage_from_stdin() {
 
 # Print out some info that isn't a top level status line
 kube::log::info() {
+  local V="${V:-0}"
+  if [[ $KUBE_VERBOSE < $V ]]; then
```

Comment:
I was trying to match the behavior of glog, which is to log if the verbosity level is >= the log level.